apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -723,6 +764,15 @@ test-k8s-ssr:
 k8s-delete-ssr:
 	${KUBECTL} delete -n ${K8S_NAMESPACE} --ignore-not-found deploy ${SSR_NAME}
 
+k8s-hpa-ssr:
+	j2 hpa.ssr.yaml.j2 | ${KUBECTL} apply -n ${K8S_NAMESPACE} -f -
+
+test-k8s-hpa-ssr:
+	j2 hpa.ssr.yaml.j2 | kubeval --strict --ignore-missing-schemas
+
+k8s-delete-hpa-ssr:
+	${KUBECTL} delete -n ${K8S_NAMESPACE} --ignore-not-found hpa ${SSR_NAME}
+
 check-service-env:
 	./check_infra_lock.sh
 
```

Comment:
I think we should also add all of these new `make` targets to `.PHONY` below.

Reply:
Added  what i think is needed here :) via 0edcdfdac15b080bb4f47604f2cc4ef1b7d69459